workspaces/analytics/src/consistentAnonymousId.ts

```diff
@@ -0,0 +1,2 @@
+import { v4 as uuidv4 } from 'uuid';
+export const consistentAnonymousId = `anonymous_user_${uuidv4()}`;
```

Comment:
might want to document here that it must start with anonymous_, or capture that in some test